### ISSUE TITLE
Multiple select share crash

### ISSUE THREAD

**Issue by licaon-kter** [mattermost/mattermost-mobile#2562]
#### Environment Information

- OS Version: Pie
- Mattermost App Version: 1.15.2
- Mattermost Server Version: 5.7.1

#### Steps to reproduce
Share multiple files from an app to Mattermost

#### Expected behavior
Show a view to select user/channel to share to...

#### Observed behavior
```
E AndroidRuntime: FATAL EXCEPTION: mqt_native_modules
E AndroidRuntime: Process: com.mattermost.rn, PID: 27660
E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
E AndroidRuntime: 	at com.mattermost.share.ShareModule.processIntent(ShareModule.java:168)
E AndroidRuntime: 	at com.mattermost.share.ShareModule.data(ShareModule.java:102)
E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
E AndroidRuntime: 	at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
E AndroidRuntime: 	at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:160)
E AndroidRuntime: 	at com.facebook.react.bridge.queue.NativeRunnable.run(Native Method)
E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:873)
E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:29)
E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:193)
E AndroidRuntime: 	at com.facebook.react.bridge.queue.MessageQueueThreadImpl$3.run(MessageQueueThreadImpl.java:192)
E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:764)
```

**Comment by amyblais** [comment]
Hi @licaon-kter, letting you know that our QAs reproduced this and opened a ticket here: https://mattermost.atlassian.net/browse/MM-14249.

**Comment by amyblais** [comment]
Closing this as this has been fixed for v1.16.1 dot release.
